virttest/qemu_devices/qcontainer.py

```diff
@@ -65,26 +68,26 @@ def get_hmp_cmds(qemu_binary):
                     hmp_cmds.extend(cmd.split('|'))
             return hmp_cmds
 
-        def get_qmp_cmds(qemu_binary, workaround_qemu_qmp_crash=False):
+        def get_qmp_cmds(workaround_qemu_qmp_crash=False):
             """ :return: list of qmp commands """
+            c1 = ('echo -e \'{ "execute": "qmp_capabilities" }\n'
+                  '{ "execute": "query-commands", "id": "RAND91" }\n'
+                  '{ "execute": "quit" }\'')
+            c2 = '(sleep 1; cat )'
+            c3 = '%s -M %s -qmp stdio -vnc none | grep return | grep RAND91' % \
+                 (self.__qemu_binary, self.__machine_type)
```

Comment:
Well, I slightly prefer having it embedded directly (for the sake of memory consumption), but it can be a bit easier to read like that. Anyway the variables would have to be named properly to express what they stand for and more importantly, do not mix style fixes with refactors, nor new features. They have to be in separate commits. (a good indication of a bad commit is when you need to use `also` or `and` in the commit message to join 2 different things)